blog-posts/gather-metaverse-bump-each-other-2021.md

```diff
@@ -0,0 +1,130 @@
+
+![](images/gather/gather-smm.jpg)
+_Our Satellytes folks gathering for our all-hands meeting on a Monday_
+
+The actual meeting is comparable to Zoom or Meet. We broadcast some news and people can share thoughts and ask questions. The time before and after that meeting is where the Metaverse shines ✨. 
+
+People bump into each other, create random groups, and tell their stories from the weekend. Compare this with the Zoom dystopia, where everyone usually joins into a single room and leave directly after the meeting back into isolation (I exaggerated a little) 
+
+### Pair programming
+A similar effect happens when you sit in the virtual office and suddenly two folks group up on the other side of the table. Instantly you know, _"oh, they do some pair programming"_. When I asked my colleagues about their session, they said it's a nice way to meet at some actual (although virtual) location and then just _"bump into the other person"_ to start the session. 
+
+This feeling is the promise of the Metaverse. It brings back the informal human interactions and spontaneity. You don't have to create a video meeting upfront or send some specific invitation. Who would do that in real life? Instead, you just walk over to the desk of your colleague and start talking.  
+
+![](images/gather/gather-pair.gif)
+
+_Our colleagues Jesús & Felix having a pair programming session_
+
+### Random bumps
+When you walk to someone, while approaching, the video and audio slow fade in. That technique is called "spatial audio technology" and mimics the way we bump into other people in real life, where someone's voice is louder while you are near them, and softer as you walk away. This is wonderful because it feels so familiar and natural.
+
+<video preload autoplay loop width="100%">
+  <source src="images/gather/video.mp4" type="video/mp4" />
+  <source src="images/gather/video.mp4" type="video/mp4" />
+</video>
+
+_Official Gather video from [gather.town](http://www.gather.town)_
+
+### Daily
+While we can gather everyone for some all-hands easily, we have the biggest struggle to motivate people to stay online throughout the day. A few colleagues and I are online most of the day. The other ones are online occasionally or just for the invited events. Sadly, this means our world looks very empty, very often.
+
+![](images/gather/gather-sy-current-world.png)
+
+_Our Satellytes space often looks that empty_
+
+What can we do about that? When I ask people, I get various feedback:
+
++ I fear I can't work focused enough because people will disturb me all the time
++ I don't trust Gather that the microphone and video is truly switched off every time I focus on another window
++ I'm intimidated with Gather as I don't play games often
++ I just forget to start Gather on the morning
+
```

Comment:
From my personal view: "It doesn't integrate in any way with my other tools" or "It requires me to spend too much focus on the app itself (the effect 'bumping' into someone is rather an active one)"

Reply:
thanks I will add those issues